docker/README.md

```diff
@@ -0,0 +1,50 @@
+# Docker
+
+<!--
+Part of the Carbon Language project, under the Apache License v2.0 with LLVM
+Exceptions. See /LICENSE for license information.
+SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+-->
+- ubuntu:20.04
```

Comment:
Maybe remove this line, or just the `-`? It seems out of place.

Reply:
Yea it does, removing it.